lib/src/generators/openapi3/openapi3.ts

```diff
@@ -349,3 +356,53 @@ function exampleToOpenApiExampleSet(
     return acc;
   }, {});
 }
+
+function contractToOa3ServerObject(
+  contract: Contract
+): ServerObject[] | undefined {
+  if (contract.oa3servers?.length === 0) {
+    return undefined;
+  }
+  const servers = contract.oa3servers;
+  const serversObject: ServerObject[] = [];
+  if (servers) {
+    for (const server of servers) {
```

Comment:
If `servers` is `undefined`, we can just return `undefined`, then we can `.map` on `servers`